index.html

```diff
@@ -0,0 +1,54 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Online Bidding</title>
+    
+    <link href="http://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,800" rel="stylesheet">
+
+    <link rel="stylesheet" href="client/css/bootstrap.css">
```

Comment:
It's better to import CSS files in js, so that you can perform various transformations using web pack at a later point, or use a cdn for better performance.